lib/rules/no-restricted-properties.js

```diff
@@ -108,6 +105,20 @@ module.exports = {
 			}
 		});
 
+		/**
+		 * Checks if an object name is in the allowed objects list.
+		 * @param {string} objectName The name of the object to check
+		 * @param {string[]} allowObjects The list of objects to allow
```

Comment:
```suggestion
		 * @param {string[]} [allowObjects] The list of objects to allow
```

As it can be undefined.